packages/plugin-session-replay-browser/src/session-replay.ts

```diff
@@ -89,10 +93,12 @@ export class SessionReplayPlugin implements EnrichmentPlugin<BrowserClient, Brow
         optOut ? 'sessionReplay.shutdown()' : 'sessionReplay.init()'
       }.`,
     );
+    // TODO: compare optOut with this.sr.getOptOut().
+    // Need to add getOptOut() to the interface AmplitudeSessionReplay first.
```

Comment:
Hi @daniel-graham-amplitude, I was planning to resolve your [comment](https://github.com/amplitude/Amplitude-TypeScript/pull/1025#discussion_r2047853213) in this PR but it requires a new API so I comment it here.